docs/source/operations/transformations/deformation.rst

```diff
@@ -105,6 +108,34 @@ Parameters
     Grids for the vertical component of a deformation model is expected to be
     in either GTX format.
 
+.. option:: +grids=<list>
+
+    .. versionadded:: 7.0.0
+
+    Comma-separated list of grids to load. If a grid is prefixed by an `@` the
+    grid is considered optional and PROJ will the not complain if the grid is
+    not available.
+
+    Grids should be in GeoTIFF format with the first 3 components being
+    respectively the easting, northing and up velocities in mm/year.
+    Setting the Description and Unit Type GDAL band metadata items is strongly
+    recommended, so that gdalinfo reports:
+
+    ::
+
+        Band 1 Block=... Type=Float32, ColorInterp=Gray
+            Description = east_velocity
+            Unit Type: mm/year
+        Band 2 Block=... Type=Float32, ColorInterp=Undefined
+            Description = north_velocity
+            Unit Type: mm/year
+        Band 3 Block=... Type=Float32, ColorInterp=Undefined
+            Description = up_velocity
+            Unit Type: mm/year
+
+    .. note:: :option:`+grids` is mutually exclusive with :option:`+xy_grids`
```

Comment:
I think it would be a good idea to include a similar note in the option descriptions for `+xy_grids` and `+z_grids`.

Reply:
Done per commit 2afa147d29631a1309e36418741e34f3f649b84b